concert/base.py

```diff
@@ -552,6 +567,10 @@ def writable(self):
         """Return True if the parameter is writable."""
         return getattr(self._instance, '_set_' + self.name) is not _setter_not_implemented
 
+    @property
+    def target_readable(self):
+        return getattr(self._instance, '_get_target_' + self.name) is not _getter_target_not_implemented
```

Comment:
Wouldn't this fail if I specified in the constructor e.g. `fget_target = 'foobarbaz'?

Reply:
I did not think about this, I just implemented it in the same ways as for the setter, which also does not work... (I will also include tests for the old stuff)